apps/ledger-live-mobile/src/components/ChangeDeviceLanguagePrompt.tsx

```diff
@@ -0,0 +1,68 @@
+import { Flex, Text, Button, Link, Icons } from "@ledgerhq/native-ui";
+import React from "react";
+import { Linking } from "react-native";
+import { useTranslation } from "react-i18next";
+import { Language } from "@ledgerhq/types-live";
+import NanoXFolded from "../images/devices/NanoXFolded";
+import { urls } from "../config/urls";
+
+type Props = {
+  onConfirm: () => void;
+  deviceName: string;
+  language: Language;
+} & (
+  | { canSkip: true; onSkip: () => void }
+  | { canSkip?: false; onSkip?: undefined }
+);
+
+const ChangeDeviceLanguagePrompt: React.FC<Props> = ({
+  deviceName,
+  language,
+  onConfirm,
+  canSkip,
+  onSkip,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Flex alignItems="center">
+      <NanoXFolded size={200} />
```

Comment:
Hardcoded nano x?

Reply:
Yes, this illustration specifically is called NanoX but is a bit generic to nano devices, it isn't clear it's a nanoX and I don't think we have an equivalent for each device yet. I'll talk to Aamir to confirm this.

Reply:
Okay!